File: web/src/lib/type/craftFavorite.ts
// POST /api/craftFavorite/{craftId}, GET /api/craftFavorite/{craftId}/status
export type CraftFavoriteStatus = {
  isFavorited: boolean;
};

// GET /api/craftFavorite/mine
export type MyCraftFavorite = {
  craftFavoriteId: string;
  craftId: string;
  craftProductName: string;
  craftAddress: string | null;
  craftPrefecture: string | null;
  craftImage: string | null;
  successorCount: number;
  createAt: string;
};

File: api/models/craftFavorite.cs
namespace Api.Model;

public class CraftFavorite
{
  public Guid CraftFavoriteId { get; set; }
  public Guid UserId { get; set; }
  public User? User { get; set; }
  public Guid CraftId { get; set; }
  public Craft? Craft { get; set; }
  public DateTime CreateAt { get; set; }
}
